Makefile

```diff
@@ -48,10 +48,9 @@ ios:
 	@echo "Done building."
 	@echo "Import \"$(GOBIN)/Geth.framework\" to use the library."
 
-test: all
-	# $(GORUN) build/ci.go test
+test:
```

Comment:
Just a note here, `go test` already does the compilation so we do not need to run `all`.